generators/newfpmpage/index.js

```diff
@@ -41,6 +41,13 @@ module.exports = class extends Generator {
                 validate: utils.validateAlhpaNumericStartingWithLetter,
                 default: 'Main'
             })).viewName;
+            this.answers.enableTypescript = (await this.prompt({
+                type: "confirm",
+                name: "enableTypescript",
+                message: "Do you want to use the awesomeness of Typescript for your controller?",
+                default: true,
```

Comment:
default `true`, hmm, I would do it vice-versa - or as this is being added on top of a project maybe this option can also be derived? Not fully sure yet...

Reply:
Yeah, default true is a bit optimistic. I will fix it and check wether it is an active ts project or not. I am working on exposing the check in the `open-ux-tools` anyhow (https://github.com/SAP/open-ux-tools/pull/667) since it is required for the SAP Fiori tools as well.

Reply:
Removed the question. I am checking if TS is enabled, and if yes, TS is used, otherwise, JS.
https://github.com/ui5-community/generator-ui5-project/pull/58/commits/0fc2353416b540d0be8eed26993cc843f726cd8b